### PR TITLE
Hotfix/sangshuduo/td 11842 split taostools targz

### PR DESCRIPTION
**Pull request by sangshuduo**

### CODE REVIEW

**Review by afwerar — APPROVED** [review]
LGTM

File: packaging/tools/install-taostools.sh [hunk 1]
```diff
@@ -72,17 +72,22 @@ function kill_process() {
...
 
+    ${csudo} /usr/bin/install -c -m 755 ${script_dir}/bin/taosdump ${install_main_dir}/bin/taosdump
+    ${csudo} /usr/bin/install -c -m 755 ${script_dir}/bin/taosBenchmark ${install_main_dir}/bin/taosBenchmark
+    ${csudo} ln -sf ${install_main_dir}/bin/taosBenchmark ${install_main_dir}/bin/taosdemo
     #Make link
-    [ -x ${install_main_dir}/bin/taosdemo ] && ${csudo} ln -s ${install_main_dir}/bin/taosdemo ${bin_link_dir}/taosdemo          || :
-    [ -x ${install_main_dir}/bin/taosdump ] && ${csudo} ln -s ${install_main_dir}/bin/taosdump ${bin_link_dir}/taosdump          || :
+    [[ -x ${install_main_dir}/bin/taosdemo ]] && ${csudo} ln -s ${install_main_dir}/bin/taosdemo ${bin_link_dir}/taosdemo           || :
+    [[ -x ${install_main_dir}/bin/taosdump ]] && ${csudo} ln -s ${install_main_dir}/bin/taosBenchmark ${bin_link_dir}/taosBenchmark || :
+    [[ -x ${install_main_dir}/bin/taosdump ]] && ${csudo} ln -s ${install_main_dir}/bin/taosdump ${bin_link_dir}/taosdump           || :
```
Comment (zhaoyanggh) [inline packaging/tools/install-taostools.sh:90]: 1

File: packaging/tools/install-taostools.sh [hunk 2]
```diff
@@ -72,17 +72,22 @@ function kill_process() {
...
+    ${csudo} rm -f ${bin_link_dir}/taosBenchmark    || :
+    ${csudo} rm -f ${bin_link_dir}/taosdump         || :
 
+    ${csudo} /usr/bin/install -c -m 755 ${script_dir}/bin/taosdump ${install_main_dir}/bin/taosdump
+    ${csudo} /usr/bin/install -c -m 755 ${script_dir}/bin/taosBenchmark ${install_main_dir}/bin/taosBenchmark
+    ${csudo} ln -sf ${install_main_dir}/bin/taosBenchmark ${install_main_dir}/bin/taosdemo
     #Make link
-    [ -x ${install_main_dir}/bin/taosdemo ] && ${csudo} ln -s ${install_main_dir}/bin/taosdemo ${bin_link_dir}/taosdemo          || :
-    [ -x ${install_main_dir}/bin/taosdump ] && ${csudo} ln -s ${install_main_dir}/bin/taosdump ${bin_link_dir}/taosdump          || :
+    [[ -x ${install_main_dir}/bin/taosdemo ]] && ${csudo} ln -s ${install_main_dir}/bin/taosdemo ${bin_link_dir}/taosdemo           || :
```
Comment (zhaoyanggh) [inline packaging/tools/install-taostools.sh:88]: should be `[[ -x ${install_main_dir}/bin/taosBenchmark ]] && ${csudo} ln -s ${install_main_dir}/bin/taosBenchmark ${bin_link_dir}/taosdemo || :` ?

File: packaging/tools/install-taostools.sh [hunk 3]
```diff
@@ -72,17 +72,22 @@ function kill_process() {
...
+    [[ -x ${install_main_dir}/bin/taosdump ]] && ${csudo} ln -s ${install_main_dir}/bin/taosBenchmark ${bin_link_dir}/taosBenchmark || :
```
Comment (zhaoyanggh) [inline packaging/tools/install-taostools.sh:89]: shoud be `[[ -x ${install_main_dir}/bin/taosBenchmark ]] && ${csudo} ln -s ${install_main_dir}/bin/taosBenchmark ${bin_link_dir}/taosBenchmark || :`

**Review by zhaoyanggh — APPROVED** [review]

File: packaging/tools/install-taostools.sh (hunk 3 above)
Comment (sangshuduo) [reply packaging/tools/install-taostools.sh]: good finding

**Pull request merged**